deploy/python/mot_sde_infer.py

```diff
@@ -362,7 +364,7 @@ def predict_image(detector, reid_model, image_list):
             else:
                 online_tlwhs, online_scores, online_ids = reid_model.predict(
                     crops, pred_dets)
-                online_im = mot_vis.plot_tracking(
+                online_im = plot_tracking(
```

Comment:
plot_tracking only for single class MOT. 
plot_tracking_dict for single and multi class MOT, but online_tlwhs, online_ids, online_scores should alse be dict .